x-pack/plugins/lens/public/xy_visualization/xy_config_panel.scss

```diff
@@ -1,3 +1,7 @@
 .lnsXyToolbar__popover {
   width: 365px;
+
+  .lnsXyToolbar__visualOption + .lnsXyToolbar__visualOption {
+    margin-top: $euiSizeL;
+  }
```

Comment:
Is this style addition necessary? As far as I'm aware, adjacent sibling `EuiFormRow` components are already styled by EUI to have a `margin-top` applied (as shown in the screenshot below). If indeed not needed, I suggest removing this style and related class additions.

![image](https://user-images.githubusercontent.com/3884767/123301777-079b8300-d4ea-11eb-9733-6bf7132d71c6.png)

Reply:
For some reason some Spacers have been iteratively added to the popup, in addition to the regular `EuiFormRow` spacing, until it reached the initial broken state.
This PR was meant to restore a state where Spacers were uniformly distributed.

Looking at other popups in the same UI it appears that they have only the `EuiFormRow` margin, which you pointed at. It makes more sense to keep only the original EUI spacing, so I'll remove the additional one from the PR.